src/Directory.Packages.props

```diff
@@ -0,0 +1,29 @@
+<Project>
+  <PropertyGroup>
+    <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
+    <CentralPackageTransitivePinningEnabled>true</CentralPackageTransitivePinningEnabled>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageVersion Include="Microsoft.SourceLink.GitHub" Version="1.0.0" />
+    <PackageVersion Include="Microsoft.CodeAnalysis.PublicApiAnalyzers" Version="3.3.4" Condition="$(Configuration) == 'Release'"/>
```

Comment:
Open question: do we need `Condition`s like this? I got this from the corresponding project files.

Reply:
Yes I don't want public api analysis in debug mode, because of the weird setup I have where internal methods in Core are public in debug mode. The reason for _that_ is that it forces me to think explicitly about which Core methods are "internal APIs" exposed to all DL implementations and which ones are for use within Core only, but it plays havoc with the analyzer in debug mode.